### PR TITLE
update to html with classes/ ids

### PR DESCRIPTION
**Pull request by amcmilne**
first commit of html with classes and ids

### CODE REVIEW

**Comment by jkole822** [comment]
Did you pull the changes before commit and push? Not that it's a big deal, but I don't see the div containing the breweries ID.

File: index.html [hunk 1]
```diff
@@ -3,9 +3,85 @@
...
+                </h1>
+                <h2 class="subtitle">
+                  ...away from that best breweries in town.
+                </h2>
+              </div>
+            </div>
+          </section>
+    </header>
     
 </body>
```
Comment (jkole822) [inline index.html]: Careful here. You have content after closing off the body.

File: index.html [hunk 2]
```diff
@@ -3,9 +3,85 @@
...
+<div class="container">
+<div class="field has-addons">
+    <div class="control">
+      <input class="input" type="text" placeholder="Find a local brewery">
+    </div>
+    <div class="control">
+      <a class="button is-primary is-bold">
```
Comment (jkole822) [inline index.html]: I would change this to a button for semantic reasons. Idk why Bulma has lists their buttons as <a> elements.

File: index.html [hunk 3]
```diff
@@ -3,9 +3,85 @@
...
+    </div>
+    <p class="location">Call the location for details</p>
+  </div>
+  <div class="field">
+    <label class="label"></label>
+    <div class="control">
+      <input class="input" type="text" placeholder="Text input">
+    </div>
+    <p class="location">Call the location for details</p>
+  </div>
```
Comment (jkole822) [inline index.html]: What is this section for? Looks like its to contain info for the individual elements. I am going to dynamically render those sections in the index.js file. If that's the case, I can copy the structure of one of your div's here and use it in index.js

File: index.html (hunk 2 above)
Comment (amcmilne) [reply index.html]: Got it.  I'll fix that when I get home from work.

File: index.html (hunk 3 above)
Comment (amcmilne) [reply index.html]: Yes, that's what it was for.  If that works for you, that's ok.  If not, I can remove that section?

File: index.html (hunk 1 above)
Comment (amcmilne) [reply index.html]: Good catch!

**Comment by amcmilne** [comment]
I thought I pulled first, but I guess I didn't.  I'll take a look when i get home.  Thanks!

File: index.html (hunk 1 above)
Comment (amcmilne) [reply index.html]: corrected.

File: index.html (hunk 2 above)
Comment (amcmilne) [reply index.html]: corrected.

**Comment by amcmilne** [comment]
made corrections  to <button> and <body> issues.

**Review by jkole822 — APPROVED** [review]
Reviewed

**Pull request merged**